test/src/http/testRequest.js

```diff
@@ -4,6 +4,7 @@ var Logger = common.require('util/logger');
 var nock = require('nock');
 var assert = require('assert');
 var mockery = require('mockery');
+var once = require('lodash.once');
```

Comment:
can't this dependency be avoided? In any case it should be added to devDependencies.